modules/search/client/views/search-sidebar-results.client.view.html

```diff
@@ -7,6 +7,8 @@
        ng-if="!search.offer && !search.loadingOffer">
     <br><br>
     <em>Choose something from the map.</em>
+    <br><br>
+    <a href="/search/members" class="btn btn-inverse-primary">Find members by name</a>
```

Comment:
Maybe I would change the text to something like: "Looking for a person you already know? <a>Find people by name here</a>"
Also, I haven't seen how it looks but maybe the btn-link looks better?
Otherwise looks good to me.